source/projects/webRadio/de/index.md

```diff
@@ -0,0 +1,136 @@
+
+//automatisch mit dem Gerät (Radio) verbinden, wenn es eingeschaltet wird
+$ [bluetooth]# trust 12:34:56:78:9A:BC
+> [CHG] Device 12:34:56:78:9A:BC Trusted: yes
+> Changing 12:34:56:78:9A:BC trust succeeded
+
+//Finally: mit dem Radio verbinden
+$ [bluetooth]# connect 12:34:56:78:9A:BC
+```
+
+## Web Radio per Autostart abspielen
+
+Damit der Aha-Effekt größer und das Ergebnis hörbar ist, haben wir das Shell-Skript um einen weiteren Command ergänzt. Mit dem Command
+```shell
+$ vlc meinePlaylist.m3u
+```
+öffnet der bereits vorinstalliert VLC-Mediaplayer die eigens erstellte Playlist "meinePlaylist". Alle darin befindlichen Dateien wie mp3-Dateien oder weitere Playlists, beispielsweise die Playlist eures Lieblings-Web-Radios, werden mit dem Command nacheinander abgespielt. In die Playlist muss demnach lediglich eine URL zu entsprechenden Web-Radiosender hinterlegt werden.
+
+**Last but not least: das Shell-Skript in den Autostart einbinden.** Um ein Programm in den Autostart des Raspberry's einzubinden gibt es verschiedene Möglichkeiten. In unserem Tutorial haben wir eine **.dektop-Datei** erstellt, um unser Skript mit jedem Hochfahren des RasPi's auszuführen. 
+```shell
+$ sudo geany /etc/xdg/autostart/NameDerDatei.desktop&
+```
+Nice To Know: durch das kaufmännische "Und" am Ende des Befehls, könnt ihr das Terminal weiterhin bedienen, obwohl der Texteditor geöffnet ist.
+Die .desktop-Dateien haben eine vorgeschriebene schematische Struktur, die es zu berücksichtigen gilt.
+```plain
+[Desktop Entry]
+Type=Application
+Name=irgendeinName
+Terminal=false
+Exec=/bin/sh /usr/bin/meinSkript.sh
+```
+Nach dem Attribut "Exec=..." wird der beim Autostart auszuführende Befehl angegeben. Da wir für die Verbindung via Bluetooth und das Abspielen des Web-Radiosenders mehrere Befehle benötigen, verlinken wir an dieser Stelle also zu unserem bereits erstellten Shell Skript "meinSkript.sh".
+
+## Ausblick
+
+Wer jetzt neugierig geworden ist und gerne selber aktive werden möchte, findet weitere Instruktionen im ausführlichen [Tutorial auf GitHub](https://github.com/technologiestiftung/werkstatt/blob/master/HowTo_WebRadio.md). Darin wird unter anderem erklärt wie:
+* die IP des RasPi's abfragt werden kann.
+* der RasPi über SSH angesteuert werden kann.
+* die .desktop-Autostart-Datei angelegt und abgelegt werden sollte.
+* der Audio-Ausgang des Raspberry eingerichtet werden kann.
+* uvm.
+
+
+Wer zu Hause etwas experimentieren möchte, jedoch keinen eigenen RasPi besitz, der kann sich **RasPi's + Zubehör kostenlos via [Hacking Box](https://www.technologiestiftung-berlin.de/hackingbox/) ausleihen.** Darüber hinaus bietet das [CityLAB Berlin](https://www.citylab-berlin.org/) monatlich stattfindende, partizipative Workshops zu Themen wie Internet of Things, Mikrokontroller, Smart Home und Hardware Hacking an. 
+
+Bei Fragen rund um das Thema *RasPi & Web Radio* hilft [Julia Zimmermann](mailto:Zimmermann@technologiestiftung-berlin.de) gerne weiter.
```

Comment:
Mails in Markdown can be written as such `<foo@bah.com>` but you don't have to.

Reply:
didn't changed it but will keep it in mind for future articles